skyvern/core/script_generations/skyvern_page.py

```
@@ -806,6 +805,7 @@ async def _ai_upload_file(
                 LOG.exception(f"Failed to adapt value for input text action on selector={selector}, file={files}")
         if not files:
             raise ValueError("file url must be provided")
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)